app/screens/InvitationsScreen.js

```diff
@@ -1,78 +1,76 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import { View, Alert, StyleSheet } from 'react-native';
 
 import { useAuth } from '../providers/AuthProvider';
-import { useDevices } from '../providers/DevicesProvider';  // USE LATER
+import { useDevices } from '../providers/DevicesProvider';
 import { useGroupManager } from '../hooks/useGroupManager';
-import { FormTextInput } from '../components/FormTextInput';
 import { List } from '../components/List';
 import { ModalForm } from '../components/ModalForm';
+import { DropdownPicker } from '../components/DropdownPicker';
 
 export function InvitationsScreen() {
   const { userData } = useAuth();
-  //const { devices } = useDevices(); // USE LATER
+  const { devices } = useDevices();
   const { respondToInvitation } = useGroupManager();
   const [selectedInvitation, setSelectedInvitation] = useState(null);
-  const [temporaryDeviceId, setTemporaryDeviceId] = useState('');
+  const [selectedPickerItem, setSelectedPickerItem] = useState(null);
+  const [pickerItems, setPickerItems] = useState([]);
+  
+  useEffect(() => {
+    setPickerItems(devices.map(device => ({ label: device.name, value: device._id.toString() }) ));
+  }, [devices.length]);
```

Comment:
```suggestion
  }, [devices.length, setPickerItems]);
```

I'm wondering if this shouldn't also depend on `setPickerItems` and perhaps `devices` instead of simply `devices.length`?